omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -313,10 +292,10 @@ Basic setup for OMERO using NGINX::
     $ export HTTPPORT=${HTTPPORT:-8080}
     $ omero web config nginx-development --http $HTTPPORT > $(brew --prefix omero53)/etc/nginx.conf
```

Comment:
Unfortunately, on snapper I am getting
```
snapper:openmicroscopy hudson$ omero web config nginx-development --http $HTTPPORT > $(brew --prefix omero53)/etc/nginx.conf
-bash: /usr/local/Cellar/omero53/5.3.3/etc/nginx.conf: No such file or directory
```
Is it maybe because the ngnix was already installed by you @will-moore, previously and thus the name was changed ?

Reply:
I think that line has never worked. I should have removed those 2 lines since they're replaced by the lines below.

Reply:
Checking the content of the ``/usr/local/etc/nginx/`` I can see the nginx.conf file there, so my previous comment is probably not valid. I guess there might be a typo in the line ?

Reply:
The only question I have is, after ```brew install nginx``` will this line work ```mv /usr/local/etc/nginx/nginx.conf /usr/local/etc/nginx/nginx.conf.orig``` (does that nginx.conf already exist by this point)?

Reply:
`brew --prefix omero53` is definitely no longer available since these recent PRs have removed dependency on that formula. Instead, `$(brew --prefix nginx)` would point to the right location.

Reply:
```$ brew --prefix nginx``` gives me ```/usr/local/opt/nginx``` which is different from ```/usr/local/etc/nginx/``` so probably just stick with what we have?

Reply:
@will-moore : Yes, the line ``mv /usr/local/etc/nginx/nginx.conf /usr/local/etc/nginx/nginx.conf.orig`` worked, as indicated above, the ``...nginx.conf`` file did already exist